src/hooks/useSelector.ts

```diff
@@ -0,0 +1,27 @@
+import { useCurrencies } from "../hooks/useCurrencies";
+import { SelectOption } from "../models";
+
+export function useSelector(
```

Comment:
что такое Selector несовсем понятно 
что делает этот хук, если мы видим его в коде? возвращает какой-то selector?
название переменной/функции должно четко и ясно отражать смысл, позволяя разработчику понимать, что делает код, не ныряя внутрь функций и хуков

```suggestion
export function useSelectorOption(
```